src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -21,12 +22,17 @@
 
 class UserPasswordValidator extends ConstraintValidator
 {
-    private $securityContext;
+    private $tokenStorage;
     private $encoderFactory;
 
-    public function __construct(SecurityContextInterface $securityContext, EncoderFactoryInterface $encoderFactory)
+    /**
+     * @param SecurityContextInterface|TokenStorageInterface
+     *
+     * Passing a SecurityContextInterface as a first argument was deprecated in 2.7 and will be removed in 3.0
+     */
+    public function __construct($tokenStorage, EncoderFactoryInterface $encoderFactory)
```

Comment:
I can only do that if I bumped the deps to 2.6 as before 2.6, the class did not extend the new interface.